paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1229,26 +1235,24 @@ def log_slo_rollback(self) -> None:
         rollback_details = self.__build_rollback_audit_details(
             RollbackTypes.AUTOMATIC_SLO_ROLLBACK
         )
-
-        dimensions = dict(rollback_details)
-        dimensions["paasta_service"] = self.service
-        self.metrics_interface.emit_event(
-            name="rollback", dimensions=dimensions,
-        )
-        _log_audit(
-            action="rollback", action_details=rollback_details, service=self.service,
-        )
+        self._log_rollback(rollback_details)
 
     def log_user_rollback(self) -> None:
         rollback_details = self.__build_rollback_audit_details(
             RollbackTypes.USER_INITIATED_ROLLBACK
         )
-
-        dimensions = dict(rollback_details)
-        dimensions["paasta_service"] = self.service
-        self.metrics_interface.emit_event(
-            name="rollback", dimensions=dimensions,
-        )
+        self._log_rollback(rollback_details)
+
+    def _log_rollback(self, rollback_details: Dict[str, str]) -> None:
+        base_dimensions = dict(rollback_details)
+        base_dimensions["paasta_service"] = self.service
+        # Emit one event per cluster to sfx
+        for cluster in self.instance_configs_per_cluster.keys():
+            dimensions = dict(base_dimensions)
```

Comment:
minor nit: is it necessary to explicitly declare it as dict()? Or is it more for readability?

Reply:
I needed to copy the base_dimensions dict rather than modify it, so opted for `dict()` here. Is there a better pattern to follow for this?  `base_dimensions.copy()`?

Reply:
Ah no no. That's correct. By adding dict(), it will make base_dimentions' own dict rather than a reference of rollback_details.